src/frontend/src/pages/ProjectDetailPage/ProjectViewContainer/ProjectViewContainer.tsx

```diff
@@ -63,6 +63,9 @@ const ProjectViewContainer: React.FC<ProjectViewContainerProps> = ({ proj, enter
       Request Change
     </MenuItem>
   );
+
+  const assignToMyTeamBtn = <MenuItem onClick={() => {}}>Assign to My Team</MenuItem>;
```

Comment:
assignToMyTeamButton -- no need to shorten names it just makes things confusing to read